numba_dpex/__init__.py

```diff
@@ -111,33 +111,30 @@ def parse_sem_version(version_string: str) -> Tuple[int, int, int]:
 from numba_dpex.dpctl_iface import _intrinsic  # noqa E402
 from numba_dpex.dpnp_iface import dpnpimpl  # noqa E402
 
-if config.HAS_NON_HOST_DEVICE:
-    # Re export
-    from .core.targets import dpjit_target, kernel_target
-    from .decorators import dpjit, func, kernel
-    from .ocl.stubs import (
-        GLOBAL_MEM_FENCE,
-        LOCAL_MEM_FENCE,
-        atomic,
-        barrier,
-        get_global_id,
-        get_global_size,
-        get_group_id,
-        get_local_id,
-        get_local_size,
-        get_num_groups,
-        get_work_dim,
-        local,
-        mem_fence,
-        private,
-        sub_group_barrier,
-    )
+from .core.targets import dpjit_target, kernel_target  # noqa E402
+from .decorators import dpjit, func, kernel  # noqa E402
+from .ocl.stubs import (  # noqa E402
+    GLOBAL_MEM_FENCE,
+    LOCAL_MEM_FENCE,
+    atomic,
+    barrier,
+    get_global_id,
+    get_global_size,
+    get_group_id,
+    get_local_id,
+    get_local_size,
+    get_num_groups,
+    get_work_dim,
+    local,
+    mem_fence,
+    private,
+    sub_group_barrier,
+)
+
+DEFAULT_LOCAL_SIZE = []
```

Comment:
Why do we still have this? We need to completely nuke the old way of setting the kernel lauch parameters.

Reply:
Right. `DEFAULT_LOCAL_SIZE` is referenced in the vectorizers.py module. The vectorizers.py is badly broken and non functional at the moment that is why I did not remove the code from `__init__.py`. I am going to do that in a follow up PR.

Reply:
Created #1273 to keep track of the pending clean up.